packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -72,10 +72,10 @@ const validProjectName = (s) => {
     return regex.test(s) || 'Value can only contain letters, numbers, space and hyphens.'
 }
 
-const validAppExtensionNameRegex = /^(@[a-zA-Z0-9-_]+\/)?extension-[a-zA-Z0-9-_]+$/
+const validAppExtensionNameRegex = /^(@[a-zA-Z0-9-_]+\/)?[a-zA-Z0-9-_]+$/
```

Comment:
I wonder if we want to use this pacakge here https://www.npmjs.com/package/validate-npm-package-name

It looks like it has a lot of usage and potentially more robust than our regex.

Reply:
Good idea. I updated the generator to use the package instead of our regex.